.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: java
```

Comment:
Would [python](https://docs.travis-ci.com/user/languages/python/) be more appropriate here?